src/pages/Main/components/Header/Header.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useRef, useState } from 'react';
+import CircleBtn from '../common/CircleBtn';
+import ImageCard from '../common/ImageCard';
+import './Header.scss';
+import DirectionBtn from '../common/DirectionBtn';
```

Comment:
```suggestion
import React, { useEffect, useRef, useState } from 'react';
import CircleBtn from '../common/CircleBtn';
import ImageCard from '../common/ImageCard';\
import DirectionBtn from '../common/DirectionBtn';
import './Header.scss';
```
import 순서 컨벤션에 맞게 수정해주세요! 사소한 부분이라 놓치기 쉽지만 이런 컨벤션 준수들이 모여서 좀 더 가독성 좋은 코드가 됩니다. 아래 순서 참고해주세요.

- 라이브러리
    - React 관련 패키지
    - 외부 라이브러리
- 컴포넌트
    - 공통 컴포넌트 → 먼 컴포넌트 → 가까운 컴포넌트
- 함수, 변수 및 설정 파일
- 사진 등 미디어 파일(`.png`)
- css 파일 (.`scss`)

Reply:
메모 해놓겠습니다 감사합니다.